framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/resource/impl/ProjectImpl.java

```diff
@@ -120,6 +124,21 @@ public void processTemplateAndCreateResources(URL templateUrl, Map<String, Strin
         }
     }
 
+    @Override
+    public void createSecret(String secretName, Map<String, String> secrets) {
```

Comment:
You can replace this implementation with https://github.com/xtf-cz/xtf/blob/master/builder/src/main/java/cz/xtf/builder/builders/SecretBuilder.java
and https://github.com/xtf-cz/xtf/blob/master/core/src/main/java/cz/xtf/core/openshift/OpenShift.java#L516

Reply:
will do!

Reply:
@Sgitario Have you tried the SecretBuilder from above? That way you should be able to get rid of encoding in here.

Reply:
Yes, I did and it's working fine. Thanks to the SecretBuilder, I also deleted the Base64Utils class that I added as part of this PR as well. 

I didn't update this PR yet because there are more missing bits: the trial tests are not working for example. Once updated, I will mark these comments as resolved, so you can review again.

Reply:
ok, thanks for info

Reply:
@sutaakar see the latest changes within the PR.

Reply:
Looks good now